cfgov/archival/forms.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import annotations
```

Comment:
What purpose does this line serve?

Reply:
Part of this is old code that has type hints. This should be removed.